docs/Configuration.md

```diff
@@ -1177,6 +1177,7 @@ export default config;
 ```
 
 We hope to support Prettier v3 seamlessly out of the box in a future version of Jest. See [this](https://github.com/jestjs/jest/issues/14305) tracking issue.
+
```

Comment:
Could you split this change into separate PR, please?

Reply:
Sure. I'll do it later.